packages/grafana-ui/src/saga-themes/tokens/$metadata.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Are we keeping the whole tokens thing in grafana/ui or moving it to a separate package?